satpy/tests/reader_tests/test_msu_gsa_l1b.py

```diff
@@ -0,0 +1,180 @@
+                    },
+                    dims=('x', 'y')),
+            'Data/resolution_4km/Solar_Zenith_Angle':
+                xr.DataArray(
+                    da.ones((num_scans, num_cols), chunks=1024,
+                            dtype=np.uint16),
+                    attrs={
+                        'scale': 0.01, 'offset': 0., 'fill_value': -999.
+                    },
+                    dims=('x', 'y')),
+            'Geolocation/resolution_4km/Latitude':
+                xr.DataArray(
+                    da.ones((num_scans, num_cols), chunks=1024,
+                            dtype=np.uint16),
+                    attrs={
+                        'scale': 0.01, 'offset': 0., 'fill_value': -999.
+                    },
+                    dims=('x', 'y')),
+            'Geolocation/resolution_4km/Longitude':
+                xr.DataArray(
+                    da.ones((num_scans, num_cols), chunks=1024,
+                            dtype=np.uint16),
+                    attrs={
+                        'scale': 0.01, 'offset': 0., 'fill_value': -999.
+                    },
+                    dims=('x', 'y')),
+            'Data/resolution_4km/Brightness_Temperature_09':
+                xr.DataArray(
+                    da.ones((num_scans, num_cols), chunks=1024,
+                            dtype=np.uint16),
+                    attrs={
+                        'scale': 0.01, 'offset': 0., 'fill_value': -999.
+                    },
+                    dims=('x', 'y')),
+        }
+        return data
+
+    def get_test_content(self, filename, filename_info, filetype_info):
+        """Mimic reader input file content."""
+        num_scans = 20
+        num_cols = 2048
+        global_attrs = {
+            '/attr/timestamp_without_timezone': '2022-01-13T12:45:00',
+            '/attr/satellite_observation_point_height': '38500.0',
+            '/attr/satellite_observation_point_latitude': '71.25',
+            '/attr/satellite_observation_point_longitude': '21.44',
+        }
+
+        data = self._get_data(num_scans, num_cols)
+
+        test_content = {}
+        test_content.update(global_attrs)
+        test_content.update(data)
+        return test_content
+
+
+class TestMSUGSABReader(unittest.TestCase):
+    """Test MSU GS/A L1B Reader."""
+
+    yaml_file = "msu_gsa_l1b.yaml"
+
+    def setUp(self):
+        """Wrap HDF5 file handler with our own fake handler."""
+        from satpy._config import config_search_paths
+        from satpy.readers.msu_gsa_l1b import MSUGSAFileHandler
+        self.reader_configs = config_search_paths(os.path.join('readers', self.yaml_file))
+        # http://stackoverflow.com/questions/12219967/how-to-mock-a-base-class-with-python-mock-library
+        self.p = mock.patch.object(MSUGSAFileHandler, '__bases__', (FakeHDF5FileHandler2,))
+        self.fake_handler = self.p.start()
+        self.p.is_local = True
+
+    def tearDown(self):
```

Comment:
Accordingly this should be
```suggestion
    def teardown(self):
```

Reply:
done!